.gitignore

```diff
@@ -3,3 +3,4 @@
 *.iml
 *.log
 target
+.DS_Store
```

Comment:
probably .idea folder also should be ignored

Reply:
".idea/*
!.idea/runConfigurations/"

Reply:
it already is ;)

Reply:
aha, do we need then  .idea/runConfigurations/MainTest___lokal_database.xml, its included in git project

Reply:
since it is exclamation mark before .idea/runConfigurations it gets included.. but I dont know why we need it